### PR TITLE
Update names in tutorial

### PR DESCRIPTION
**Pull request by yujokang**
In tutorial, changed output file names to current ones,
updating Docker instructions so that the names match,
and fixed a typo for the flag "-max_total_time=".

### CODE REVIEW

**Comment by yujokang** [comment]
I signed it!

File: tutorial/libFuzzerTutorial.md [hunk 1]
```diff
@@ -40,11 +40,27 @@ Fuzzer/build.sh
...
+```shell
+# Update repositories
+cd ~/FTS
+git pull
+cd ~/Fuzzer
+svn update
+cd
+# Update packages
+apt-get update
+# Follow the installation and building steps like the previous option
```
Comment (dnoursi) [inline tutorial/libFuzzerTutorial.md]: The commands to install deps, clang, and the fuzzer are already in the Dockerfile in this directory, which makes these calls redundant

Comment (dnoursi) [inline tutorial/libFuzzerTutorial.md:54]: The commands to install deps, clang, and the fuzzer are already in the Dockerfile in this directory, which would make these calls redundant

**Review by dnoursi — COMMENTED** [review]
I had one comment on some Docker related code, and all of the rest which modify my update yesterday LGTM, @kcc

File: tutorial/libFuzzerTutorial.md (hunk 1 above)
Comment (yujokang) [reply tutorial/libFuzzerTutorial.md:54]: The image that you pull using the existing instructions is from the Docker registry, not Github. It is older, and does not include the most recent versions of the repositories and packages.

Comment (dnoursi) [reply tutorial/libFuzzerTutorial.md:54]: That's exactly right... Looking at the tutorial again, `tutorial/Dockerfile` doesn't seem to be built in any instructions

Comment (kcc) [reply tutorial/libFuzzerTutorial.md:54]: That's wrong. I'll need to rebuild the docker image instead.

**Comment by kcc** [comment]
I've rebuilt the docker images, please try. 
I've also made some changes in the scripts, e.g. I've switched the clang version from the one used by chrome to a vanilla 4.0.1 release of LLVM (fresh enough and stable)

**Comment by yujokang** [comment]
The image is updated now.
But some of the typos in the instructions did not get fixed.

**Comment by kcc** [comment]
Could you please send a separate PR (or update this one) to only address the typos? 
Thanks!

**Pull request closed without merging**